docs/reset-password.md

```diff
@@ -0,0 +1,54 @@
+# Reset TechPass password
+
+If you are a vendor and if your TechPass domain is ```techpass.gov.sg```, you may reset your TechPass password according to the [Azure Password Policy](https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences) or if your TechPass account gets locked.
+
+>**Notes for public officers**
+>
+>- If you are a public officer, and if your TechPass account gets locked after continuous unsuccessful login attempts, reset the password of your GSIB device by following the WOG password policies. 
+>
+>- If there are any issues, contact your Agency Facility Management (AFM).
+
+**To reset TechPass password**
+
+1. Go to [reset password](https://passwordreset.microsoftonline.com/) and  enter your TechPass user name.
+
+> **Note**
+>- If the email address you specified while requesting for a TechPass account, is a Microsoft account, you can either enter that email address or your TechPass user name.
```

Comment:
Our users should use either the WoG AAD email ID or their corporate email. Should we even mention using a Microsoft account?

Reply:
@JLeeChan , I think we have clearly stated that these instructions are applicable only for vendors, so IMO, there is no need to mention about WOG AAD email address.

Reply:
in my mind Microsoft account means using **outlook.com**, **microsoft.com** or maybe **hotmail.com** 
are you referring to these?

Reply:
Yes to the first 2, but i did not have Hotmail in my mind. Are we allowing that ?